deployment/CP05MOAS-GL379_Deploy.csv

```diff
@@ -64,4 +64,10 @@ TS20210131,,TS20210419,,CP05MOAS-GL379-01-ADCPAM000,11,1,2021-01-31T17:57:00,202
 TS20210131,,TS20210419,,CP05MOAS-GL379-02-FLORTM000,11,1,2021-01-31T17:57:00,2021-04-19T18:14:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-FLORTM-05133,39.92648,-71.17723,,1000,1000,SS-2,,
 TS20210131,,TS20210419,,CP05MOAS-GL379-03-CTDGVM000,11,1,2021-01-31T17:57:00,2021-04-19T18:14:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-CTDGVM-09453,39.92648,-71.17723,,1000,1000,SS-2,,
 TS20210131,,TS20210419,,CP05MOAS-GL379-04-DOSTAM000,11,1,2021-01-31T17:57:00,2021-04-19T18:14:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-DOSTAM-00171,39.92648,-71.17723,,1000,1000,SS-2,,
-TS20210131,,TS20210419,,CP05MOAS-GL379-05-PARADM000,11,1,2021-01-31T17:57:00,2021-04-19T18:14:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-PARADM-50277,39.92648,-71.17723,,1000,1000,SS-2,,
\ No newline at end of file
+TS20210131,,TS20210419,,CP05MOAS-GL379-05-PARADM000,11,1,2021-01-31T17:57:00,2021-04-19T18:14:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-PARADM-50277,39.92648,-71.17723,,1000,1000,SS-2,,
+TS20210517,,,,CP05MOAS-GL379-00-ENG000000,12,1,2021-05-17T20:48:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGCON-GLDRCP-00379,39.81772,-70.39215,,1000,1000,SS-2,,
+TS20210517,,,,CP05MOAS-GL379-01-ADCPAM000,12,1,2021-05-17T20:48:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-ADCPAM-49697,39.81772,-70.39215,,1000,1000,SS-2,,
+TS20210517,,,,CP05MOAS-GL379-02-FLORTM000,12,1,2021-05-17T20:48:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-FLORTM-04201,39.81772,-70.39215,,1000,1000,SS-2,,
+TS20210517,,,,CP05MOAS-GL379-03-CTDGVM000,12,1,2021-05-17T20:48:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-CTDGVM-09347,39.81772,-70.39215,,1000,1000,SS-2,,
+TS20210517,,,,CP05MOAS-GL379-04-DOSTAM000,12,1,2021-05-17T20:48:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-DOSTAM-00566,39.81772,-70.39215,,1000,1000,SS-2,,
+TS20210517,,,,CP05MOAS-GL379-05-PARADM000,12,1,2021-05-17T20:48:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-PARADM-50304,39.81772,-70.39215,,1000,1000,SS-2,,
```

Comment:
QLR deploy table lists SS-1 for assigned line (but the main part of the QLR lists SS-2.  Need to verify which is correct.  The rest looks ok

Reply:
379 is on the SS-2 line. Deploy CSV correct. fixed QLR.

Reply:
roger that!  looks good then.

Reply:
Looks good.